### DISCUSSION TITLE
WHY BOTHER!!!

### DISCUSSION THREAD

**Discussion by Trickiy** [3kh0/ext-remover#969] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
It's possible to use the classdojo app on Chromebooks as a web browser without gogardian and has the abuility to download files(will have to rename some of the files so they work sometimes not).

 How to do: 1.( open classdojo then click on I am a teacher
 2.( click create teacher account
 3.(click terms of service/Privacy policy both work
 4.(scroll to the bottom of the page and click the get it on Google play button
 5.(scroll down to the bottomed of the page then click on the word privacy 
 6.(click on the square made of dots and click on search.

 With the important info out of the way would it be possible to use classdojo as a means of jailbreaking? Or install a vitual machine that overrides the OS or side load apks?

**Comment by Trickiy** [comment]
Why bother sharing a big entry point into jailbreaking the Chromebook, when nobody cares.

**Reply by nathblade16** [reply, marked as answer]
Hey, I'm sorry that nobody cares, Just so you don't feel frustrated, I will clear up some things for you:
1. no this cannot be used to sideload an app, to sideload an app on Android you need more than just a web browser, the current arc setup does not allow sideloading unless you get access to ADB debugging
2. if you have devmode you can sideload
3. the most you can do with this is have a webview
I hope this clears up some stuff and no this is unfortunately not a valid entry point, one app is not enough to have a real entry point into arc

**Reply by Trickiy** [reply]
Thx for making things more clear  I feel a bit better now. But I still feel like there could still be something done within the sandbox that classdojo uses. Like getting the app to auto boot into the Google search engine when opening the app. Or overwrighting some of the code to do other things that were not originally intended. Oh and before I forget I think it is worth mentioning that class dojo probobly uses the Linux Virtual Machine: arcvm, as when I end the process in task manager classgojo closses. While the chrome browser stays open. maybe somthing could be done within the vm. downloads probobly could be redirected  as when downloads are set to download to Google drive by default classdojo ignores it.

**Reply by nathblade16** [reply]
Yes, this can be used to open Google in a webview, this is possible, if you still want an entry point, there is one in the RMA shims of Chromebooks that have been leaked, I don't really know how to explain how it is used but it is an entry point

**Reply by Trickiy** [reply]
Thx but I am on the latest update for my Chromebook so it is patched I think, so it won't work. Someone like myself need to use 
a different way.

**Reply by nathblade16** [reply]
Unless you don't have a leaked rma shim, you can do it

**Reply by Trickiy** [reply]
on v122? I thought I need to defog it and it is patched I believe.

**Reply by AshtonDavies** [reply]
> on v122? I thought I need to defog it and it is patched I believe.

Defogging is theoretically patched due to the downgrade patch, but it's not confirmed as far as I know.

Testing and confirming it would be appreciated.

**Reply by Trickiy** [reply]
Well it's kinda scary as if it dosn't work I will be locked my Chromebook and I don't have a second device so I can't try for a long while.

**Comment by nathblade16** [comment]
> on v122? I thought I need to defog it and it is patched I believe.

they only patched the unenrollment part, the entry point still exists

**Reply by Trickiy** [reply]
so your saying I could do something similar to a virtual machine os override or install a custom kiosk app essentially.

**Reply by nathblade16** [reply]
probably
